CharacterCreation/03_Feats.txt

```diff
@@ -351,6 +351,23 @@ buzzed or hammered, you deal extra damage equal to your strength modifier. These
 bonuses only apply to melee attacks. You also get a +10% on rolls to avoid 
 blacking out.  
 
+== Point Blank ==
+
+Prerequisites:
+
+*  Backstab feat
+*  Profeciency with Pistols or SMGs
```

Comment:
sp: Proficiency

Reply:
But also, since we require Proficiency to use a class of firearms at all, a proficiency prereq is a little redundant.